client/.env.local

```
@@ -1,5 +1,8 @@
+VITE_BACKEND_URL=http://localhost:3000
 VITE_PUBLIC_VRF_PROVIDER_ADDRESS=0x051fea4450da9d6aee758bdeba88b2f665bcbf549d2c61421aa724e9ac0ced8f
 VITE_PUBLIC_CLOUDFLARE_ID=z4845v01tb3yh1uw
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_PUBLIC_CLOUDFLARE_ID key should go before the VITE_PUBLIC_VRF_PROVIDER_ADDRESS key

(UnorderedKey)